sparql/flip_mondo_mappings.ru

```diff
@@ -0,0 +1,20 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix IAO: <http://purl.obolibrary.org/obo/IAO_>
+prefix MONDO: <http://purl.obolibrary.org/obo/MONDO_>
+prefix RO: <http://purl.obolibrary.org/obo/RO_>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+prefix def: <http://purl.obolibrary.org/obo/IAO_0000115>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+INSERT {
+    ?omim_id skos:exactMatch ?mondo_id .
```

Comment:
I see, we're adding some mappings in the flipped order.

Technically should this file be named `add_flipped_mondo_mappings`? `flip_` makes it sound like the triples are being replaced.

Reply:
If you think this is important, feel free to apply the change, but ensure that the makefile code is also updated accordingly

Reply:
[Done](https://github.com/monarch-initiative/omim/pull/107/commits/479cd3527528550956fb3ec15a051e4fca967699)!